Django-1.10: Replace IPAddressField with GenericIPAddressField

# esmond_manage migrate
cassandra_db [INFO] Checking/creating column families
cassandra_db [INFO] Schema check done
cassandra_db [DEBUG] Opening ConnectionPool
cassandra_db [INFO] Connected to ['localhost:9160']
SystemCheckError: System check identified some issues:

ERRORS:
api.IfRef.ipAddr: (fields.E900) IPAddressField has been removed except for support in historical migrations.
        HINT: Use GenericIPAddressField instead.
api.LSPOpStatus.dstAddr: (fields.E900) IPAddressField has been removed except for support in historical migrations.
        HINT: Use GenericIPAddressField instead.
api.LSPOpStatus.srcAddr: (fields.E900) IPAddressField has been removed except for support in historical migrations.
        HINT: Use GenericIPAddressField instead.

--- a/esmond/api/migrations/0001_initial.py
+++ b/esmond/api/migrations/0001_initial.py
@@ -101,7 +101,7 @@
                 ('ifDescr', models.CharField(max_length=512, db_column=b'ifdescr')),
                 ('ifName', models.CharField(max_length=512, db_column=b'ifname')),
                 ('ifAlias', models.CharField(max_length=512, null=True, db_column=b'ifalias', blank=True)),
-                ('ipAddr', models.IPAddressField(null=True, db_column=b'ipaddr', blank=True)),
+                ('ipAddr', models.GenericIPAddressField(null=True, db_column=b'ipaddr', blank=True)),
                 ('ifSpeed', models.BigIntegerField(null=True, db_column=b'ifspeed', blank=True)),
                 ('ifHighSpeed', models.BigIntegerField(null=True, db_column=b'ifhighspeed', blank=True)),
                 ('ifMtu', models.IntegerField(null=True, db_column=b'ifmtu', blank=True)),
@@ -143,8 +143,8 @@
             fields=[
                 ('id', models.AutoField(verbose_name='ID', serialize=False, auto_created=True, primary_key=True)),
                 ('name', models.CharField(max_length=128)),
-                ('srcAddr', models.IPAddressField()),
-                ('dstAddr', models.IPAddressField()),
+                ('srcAddr', models.GenericIPAddressField()),
+                ('dstAddr', models.GenericIPAddressField()),
                 ('state', models.IntegerField()),
                 ('begin_time', models.DateTimeField()),
                 ('end_time', models.DateTimeField(default=datetime.datetime(9999, 12, 29, 23, 59, 59, 999999, tzinfo=utc))),
--- a/esmond/api/models.py
+++ b/esmond/api/models.py
@@ -231,7 +231,7 @@
     ifName = models.CharField(max_length=512, db_column="ifname")
     ifAlias = models.CharField(max_length=512, db_column="ifalias", blank=True,
             null=True)
-    ipAddr = models.IPAddressField(blank=True, db_column="ipaddr", null=True)
+    ipAddr = models.GenericIPAddressField(blank=True, db_column="ipaddr", null=True)
     ifSpeed = models.BigIntegerField(db_column="ifspeed", blank=True, null=True)
     ifHighSpeed = models.BigIntegerField(db_column="ifhighspeed", blank=True,
             null=True)
@@ -364,8 +364,8 @@
     """Metadata about MPLS LSPs."""
     device = models.ForeignKey(Device, db_column="deviceid")
     name = models.CharField(max_length=128)
-    srcAddr = models.IPAddressField()
-    dstAddr = models.IPAddressField()
+    srcAddr = models.GenericIPAddressField()
+    dstAddr = models.GenericIPAddressField()
     state = models.IntegerField()
 
     begin_time = models.DateTimeField()
